### PR TITLE
blank file

### PR DESCRIPTION
**Pull request by tvigilante**

### CODE REVIEW

File: migration.py [hunk 1]
```diff
@@ -1,125 +0,0 @@
...
-
-##HEADER AUTHENTICATION VALUES##
-
-
-
-CUSTOMER_CODE = "hang"
-
-TARGET_SKU = "VCX-ENT"
-
-LOCATIONS_GET_URL = 'https://portal.qa.thinkingphones.com/tpn-config-rs-external/api/v1/customers/' + CUSTOMER_CODE + '/locations?maxRows=1000account='
```
Comment (charbergFuze) [inline migration.py]: These will need to be changed to production

File: migration.py [hunk 2]
```diff
@@ -1,125 +0,0 @@
...
-SERVICES_GET_URL = 'https://portal.qa.thinkingphones.com/tpn-config-rs-external/api/v1/customers/' + CUSTOMER_CODE + '/services?maxRows=1000&locationId='
-SERVICES_PUT_URL = 'https://portal.qa.thinkingphones.com/tpn-config-rs-external/api/v1/customers/' + CUSTOMER_CODE + '/services/'
-USERS_GET_URL = 'https://portal.qa.thinkingphones.com/tpn-config-rs-external/api/v1/customers/' + CUSTOMER_CODE + '/users'
-USERNAME_GET_URL = 'https://portal.qa.thinkingphones.com/tpn-config-rs-external/api/v1/customers/' + CUSTOMER_CODE + '/users?username='
-USERS_POST_URL = 'https://portal.qa.thinkingphones.com/tpn-config-rs-external/api/v1/customers/' + CUSTOMER_CODE + '/users'
-DEPARTMENTS_GET_URL = 'https://portal.qa.thinkingphones.com/tpn-config-rs-external/api/v1/customers/' + CUSTOMER_CODE + '/departments'
-
-
-def queryServices(accountCode, voiceSku, apiToken):
```
Comment (tvigilante) [inline migration.py]: we also need an apikey if it is going through kong

File: migration.py [hunk 3]
```diff
@@ -1,125 +0,0 @@
-import requests
-import json
-import sys
-
-##HEADER AUTHENTICATION VALUES##
```
Comment (charbergFuze) [inline migration.py]: Need to remove this comment

File: migration.py [hunk 4]
```diff
@@ -1,125 +0,0 @@
...
-
-
-
-CUSTOMER_CODE = "hang"
-
-TARGET_SKU = "VCX-ENT"
-
-LOCATIONS_GET_URL = 'https://portal.qa.thinkingphones.com/tpn-config-rs-external/api/v1/customers/' + CUSTOMER_CODE + '/locations?maxRows=1000account='
-SERVICES_GET_URL = 'https://portal.qa.thinkingphones.com/tpn-config-rs-external/api/v1/customers/' + CUSTOMER_CODE + '/services?maxRows=1000&locationId='
```
Comment (tvigilante) [inline migration.py]: missing 429 handling

File: migration.py [hunk 5]
```diff
@@ -1,125 +0,0 @@
...
-
-        userless[i]['userId'] = response.json()['data']['userId']      #Assign the id of the created user to the service object
-
-        response = requests.put(SERVICES_PUT_URL+str(userless[i]['id']), headers=headerVals, data=json.dumps(userless[i]))    #Update service with the assigned user id
-
-        if response.status_code != 200:
-            failedServices.append(service['id'])
-            continue
-
-if __name__ == "__main__":  #Main method
```
Comment (tvigilante) [inline migration.py]: i hate everything about this

Comment (charbergFuze) [reply migration.py]: :snake:

Comment (tmacdonald) [reply migration.py]: This code is amazing

**Pull request closed without merging**